gax/src/test/java/com/google/api/gax/retrying/ExponentialRetryAlgorithmTest.java

```diff
@@ -57,6 +58,20 @@
           .build();
   private final ExponentialRetryAlgorithm algorithm =
       new ExponentialRetryAlgorithm(retrySettings, clock);
+  private final RetrySettings retrySettingsOverride =
+      RetrySettings.newBuilder()
+          .setMaxAttempts(3)
+          .setInitialRetryDelay(Duration.ofMillis(2L))
+          .setRetryDelayMultiplier(3.0)
+          .setMaxRetryDelay(Duration.ofMillis(18L))
+          .setJittered(false)
```

Comment:
feels like this should be true, or even more likely unconditionally true; not a setting at all

Reply:
I feel that such a change is outside the scope of this feature. `RetrySettings` are used all over the place, so removing this property in this change is out of the question. Not respecting this specific property of `RetrySettings` for this specific feature also seems strange, as they are currently respected for the `globalSettings`, and also generally everywhere else where `RetrySettings` are used.